global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "5.0.100-rc.1.20452.10"
+    "version": "5.0.100-rc.2.20480.7"
```

Comment:
The officially released RC.2 sdk is `5.0.100-rc.2.20479.15`, see https://dotnet.microsoft.com/download/dotnet/5.0

Reply:
good catch fixing it.